src/assets/wise5/services/projectTagService.ts

```diff
@@ -38,10 +38,12 @@ export class ProjectTagService {
     });
   }
 
-  createTag(tagName: string): Subscription {
-    return this.http.post(`/api/user/tag`, { text: tagName }).subscribe((tag: Tag) => {
-      this.newTagSource.next(tag);
-    });
+  createTag(tagName: string): Observable<Object> {
```

Comment:
This should return a Tag Observable?
```suggestion
  createTag(tagName: string): Observable<Tag> {
```